packages/core/scss/mixins/_focus-indicator.scss

```
@@ -0,0 +1,7 @@
+@mixin focus-indicator( $indicator, $inset: false, $themeable: false, $type: "box-shadow" ) {
+    @if $kendo-enable-focus-contrast and nth($indicator, 1) {
```

We have new function

```suggestion
    @if $kendo-enable-focus-contrast and k-list-nth($indicator, 1) {
```